apps/fabric-website/src/SiteDefinition/SiteDefinition.pages/Controls/web.tsx

```diff
@@ -13,7 +13,7 @@ export interface ICategory {
 // Exporting this object to be used in generating a TOC (table of content) for docs.microsoft documentation repo.
 // Any changes to this object need to be communicated to avoid accidental breaking of the documentation
 // and to allow the appropriate actions to be taken to mitigate this.
-export const categories: { Other?: ICategory; [name: string]: ICategory } = {
+export const categories: { [name: string]: ICategory } = {
```

Comment:
This might be a better workaround:
```suggestion
export const categories: { Other?: ICategory; [name: string]: ICategory | undefined } = {
```